src/ngAnimate/animateCss.js

```diff
@@ -261,7 +261,7 @@ function computeCssStyles($window, element, properties) {
       }
 
       // by setting this to null in the event that the delay is not set or is set directly as 0
-      // then we can still allow for zegative values to be used later on and not mistake this
+      // then we can still allow for negative values to be used later on and not mistake this
```

Comment:
But from what I understand, zero values are being set to `null`, right? So in that case zero is not allowed :wink: